### ISSUE TITLE
Accessibility: MAS40B: Inspect: Name property is empty for the list item buttons

### ISSUE THREAD

**Issue by wiproaccess** [wiproaccess/vscode-website#29]
#A11yMAS
#MAS40B
#Wipro
#vscode-website

**Testing Environment:**
OS-RS1
OS Name Microsoft -Windows 10 Pro
Version-10.0.14393 Build 14393.607
Browser-Edge
Edge Version-Microsoft Edge 38.14393.607.0

**Repro Steps:**
**Scenario-1:**
1. Open [https://code.visualstudio.com/](https://code.visualstudio.com/) in EDGE.
2. Click on the list item buttons available below the tweets (with mouse as keyboard is not accessible)
3. Observe the "Name" property in Inspect(for both the buttons)
**_Same issue is also observed with the link:_**
[https://code.visualstudio.com/docs](https://code.visualstudio.com/docs)

**Scenario-2:**
Same issue is observed with the list [Refer screenshot]
**_Same issue is also observed with the link:_**
[https://code.visualstudio.com/docs
](https://code.visualstudio.com/docs
)

**Actual Result:**
**Scenario-1:**
The "Name" property is empty.
**_Narrator Behavior:_**
Narrator announces as "One of Two"/ "Two of Two"

**Scenario-2:**
The "Name" property is empty.
**_Narrator Behavior:_**
Narrator announces as "List"

**Expected Result:**
The "Name" property should be specific

**User Impact:**
Visually impaired users  will not get to know the purpose of the link as the screen reader will not announce anything when the user arrives at the control.
Providing the name property enables the compatibility with the Screen Reader, which may be used by people with disabilites.

**Screenshots:**
![name property is empty for the whole list](https://cloud.githubusercontent.com/assets/24825441/21709632/83dce1e4-d408-11e6-8331-92379dda048d.png)
![name proprty is empty for the list item buttons](https://cloud.githubusercontent.com/assets/24825441/21709633/83dd7492-d408-11e6-86d0-046204f0d9b4.png)

**Comment by waderyan** [comment]
This element has been removed.
